File: Poin A/Processor.cpp
#include<iostream>

using namespace std;

class Processor{
    private :
        //atribut
        string name;
        int priceP;
    public :
        //konstruktor
        Processor(){}
        Processor(string n, int p){
            this->name = n;
            this->priceP = p;
            
        }

        //setter & getter
        void setName(string n){
            this->name = n;
        }

        void setPriceP(int p){
            this->priceP = p;
        }

        string getName(){
            return this->name;
        }

        int getPriceP(){
            return this->priceP;
        }


        //output
        void printProcessor(){
            cout << "Processor    : " << this->getName() << endl;
            cout << "Harga        : " << this->getPriceP() << endl;


        }

        ~Processor(){}

};

File: Poin B.php/Processor.php
<?php
class Processor
{
    //atribute
    private $name;
    private $price;

    //constructor
    function __construct($name, $price)
    {
        $this->name = $name;
        $this->price = $price;
    }

    // get set methods
    function setName($name)
    {
        $this->name = $name;
    }
    function  getName()
    {
        return $this->name;
    }
    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }

    // Output Methdos
    function output()
    {
        echo "Processor                 : " . $this->getName() . "<br>";
        echo "Harga                     : " . $this->getPrice() . "<br>";
    }

    //destructor
    function __destruct()
    {
    }
}